src/main/java/no/nav/foreldrepenger/oversikt/domene/svp/DtoMapper.java

```diff
@@ -98,14 +99,12 @@ private static no.nav.foreldrepenger.common.innsyn.svp.Arbeidsforhold tilDto(Til
 
     private static Set<no.nav.foreldrepenger.common.innsyn.svp.Tilrettelegging> fjernPerioderMedOpphold(Set<no.nav.foreldrepenger.common.innsyn.svp.Tilrettelegging> tilretteleggingDtos,
                                                                                                         Set<no.nav.foreldrepenger.common.innsyn.svp.OppholdPeriode> oppholdDtos) {
-        var oppholdSegments = oppholdDtos.stream()
+        var timelineOpphold = oppholdDtos.stream()
+            .map(p -> new LocalDateSegment<>(p.fom(), p.tom(), Boolean.TRUE))
+            .collect(Collectors.collectingAndThen(Collectors.toList(), l -> new LocalDateTimeline<>(l, StandardCombinators::alwaysTrueForMatch)));
```

Comment:
jeg lurer litt på om vi gir informasjon til bruker om kilde? For disse periodene hadde ulike kilde..

Og hvordan ville denne håndtert det om den ene perioden er innenfor den andre? Blir det en periode da?

Reply:
I oversikt vises det som "Du har ferie" uten henvisning til kilde: https://navikt.github.io/foreldrepengesoknad/foreldrepengeoversikt/index.html?path=/story/svp-oversikt--case-1

Reply:
@AnjaAalerud da må du se på koden litt lenger opp. Linje 54 + 92 samler alle opphold med kilde
Har ikke rørt disjoint og oppholdkilde ble ikke brukt i denne metoden - som viser perioder med uttak